linux/packaging/deb/make_config.yaml

```diff
@@ -1,47 +0,0 @@
-display_name: Spotube
-package_name: spotube
-
-maintainer:
-  name: Kingkor Roy Tirtho
-  email: krtirtho@gmail.com
-
-priority: optional
-section: x11
-installed_size: 24400
-
-dependencies:
-  - mpv
-  - libappindicator3-1 | libayatana-appindicator3-1
-  - gir1.2-appindicator3-0.1 | gir1.2-ayatanaappindicator3-0.1
-  - libsecret-1-0
-  - libnotify-bin
-  - libjsoncpp25
-  - libmpv1 | libmpv2
-  - xdg-user-dirs
-  - avahi-daemon
-  - avahi-discover
-  - avahi-utils
-  - libnss-mdns
-  - mdns-scan
-  - libwebkit2gtk-4.1-0 | libwebkit2gtk-4.0-0
-  - libsoup-3.0-0 | libsoup-2.4-0
-
-suggested_dependencies:
-  - yt-dlp
-
-essential: false
-icon: assets/spotube-logo.png
-
-keywords:
-  - Music
-  - Spotify
-  - Media
-  - Streaming
-  - YouTube
-
-generic_name: Music Streaming Application
-categories:
-  - Music
-
-supported_mime_type:
-  - x-scheme-handler/spotify
```

Comment:
Again, why the necessity to remove this file? It's important to generate a `.deb ` for linux platform